testplan/common/remote/remote_service.py

```diff
@@ -114,6 +125,19 @@ def starting(self):
             cwd=self.runpath,
         )
 
+        self.logger.debug(
+            "%s executes cmd: %s\n"
+            "\tRunpath: %s\n"
+            "\tPID: %s\n"
+            "\tOut file: %s\n"
+            "\tErr file: %s",
+            self,
+            cmd,
+            self.runpath,
+            self.proc.pid,
+            self.std.out_path,
+            self.std.err_path,
+        )
         self.logger.debug(
```

Comment:
I guess this need to go now in favor of the above

Reply:
Disagree on this one. The last time we voted for lazy logging. Consistency, I think is more important, than nicely looking function call. A solution in the middle is to modify the message so that it wants less in a single go. In any case, I would stick to lazy logging unless we go full on f-strings/format method.

Reply:
I just tried pointing out that you did not removed the fstring version of the log, if I see it correctly on github.

Reply:
Ah... sorry, let me remove that. :S